### PR TITLE
Lite intallation

### PR DESCRIPTION
**Pull request by kfir4444**
Added a lite installation feature, managed to drop the arc's size from ~190MB to ~80MB.

### CODE REVIEW

File: devtools/lite.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# delete all the Test and unnecessary files
+pushd .
```
Comment (alongd) [inline devtools/lite.sh:3]: When using `pushd .` (storing the current path), I think we should end the script with `popd || exit` (returning to the original path)

File: devtools/lite.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+rm -r functional
```
Comment (alongd) [inline devtools/lite.sh:5]: Should we add `-f` (so `-rf` actually) to ignore nonexisting files? Might be useful if a user executed this script, stopped it for some reason, and then run it again

File: devtools/lite.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+rm -r ipython
+rm -r arc/testing
+
+find "$PWD" -path '*Test.py' -type f -delete
```
Comment (alongd) [inline devtools/lite.sh:9]: What does this do?

**Review by alongd — COMMENTED** [review]
Thanks!

File: devtools/lite.sh (hunk 2 above)
Comment (kfir4444) [reply devtools/lite.sh:5]: good point!

File: devtools/lite.sh (hunk 3 above)
Comment (kfir4444) [reply devtools/lite.sh:9]: This finds all the files of the type `*Test.py` in the current working directory, and delete them.

File: devtools/lite.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+# delete all the Test and unnecessary files
+pushd .
+
+rm -rf functional
+rm -rf ipython
+rm -rf arc/testing
+
+find "$PWD" -path '*Test.py' -type f -delete
+
+popd || exit
```
Comment (alongd) [inline devtools/lite.sh:11]: Minor: please finish scripts with an empty line

**Review by alongd — COMMENTED** [review]
Looking good, added a minor comment. Would you like to add all the `make` option to the docs?

**Review by alongd — APPROVED** [review]
Thanks!

**Pull request merged**